src/site/content/en/media/prepare-media/index.md

```diff
@@ -0,0 +1,75 @@
+---
+layout: post
+title: Prepare media files for the web
+authors:
+  - joemedley
+description: |
+  Take a raw video file off a camera and transform it into a resource that you
+  can embed in a web page.
+date: 2017-06-30
+updated: 2020-08-20
+tags:
+  - FFmpeg
+  - files
+  - Shaka
+---
+
+Now that I've introduced you to [applications used for manipulating media
+files](../application-basics), over the next few pages, I'm going to take a raw
+video file off a camera and transform it into a resource that you can
+embed in a web page. I'm specifically going to show you how to format your video
+for mobile playback, and how to create multiple files to cover a range of
+browsers. Specifically, I'll create a webm file for use on Chrome and an mp4
+file for use on other browsers.
+
+{% Aside %}
+This section provides explanations of file manipulation concepts,
+with command lines only to illustrate the concepts. There is a companion [cheat
+sheet](../cheatsheet) that shows more commands and is designed as a quick reference
```

Comment:
```suggestion
sheet](/media-cheatsheet) that shows more commands and is designed as a quick reference
```
I haven't reviewed the cheat sheet page yet but we're going to need a different URL